### PR TITLE
notification drawer

### PR DESCRIPTION
**Pull request by cliffpyles**
Issue: https://github.com/patternfly/patternfly-next/issues/35

### CODE REVIEW

**Comment by cliffpyles** [comment]
Issue: https://github.com/patternfly/patternfly-next/issues/35

File: src/_site/Example/index.js [hunk 1]
```diff
@@ -25,13 +25,13 @@ export default ({heading, description, children}) => {
...
         <h4 className="Example__section_heading">Preview</h4>
-        <div className="Example__section__body">
+        <div className="Example__section__body is-preview">
           {children}
         </div>
       </div>
       <div className="Example__section">
         <h4 className="Example__section_heading">Code</h4>
-        <div className="Example__section__body has-no-padding">
+        <div className="Example__section__body has-no-padding is-code">
```
Comment (andresgalante) [inline src/_site/Example/index.js]: I am confused by some parts of this PR like this one, I though we were not doing utility classes. Is that the rule?

Comment (cliffpyles) [reply src/_site/Example/index.js]: This component isn't part of the library.

**Comment by andresgalante** [comment]
Hi Cliff, this comment is more to understand the system rather than request for changes, and if you have time I'd love to talk about it:

Why is the media object a component and not a layout if all it does is position it's children? and why not using a split layout if it does the same as a media object?

Would the drawer be "an absolute positioned container layout that comes from either the right or the left" and then within that drawer layout we would place a box with the accordion inside?

**Comment by cliffpyles** [comment]
I was originally thinking they would have slight differences(media object and a split layout), specifically they media-object could have visible styles applied to it, like borders, padding, colors, etc.

Yes. We could also take that approach to the drawer, but in the end we would still need to create a drawer component to account for the non-layout related part of the component.

@andresgalante Are we good with merging this as a demo for the meetup?

**Comment by LHinson** [comment]
@cliffpyles yes, after talking with you and @andresgalante. Let's close it. Thx!

**Pull request closed without merging**